pages/index.vue

```diff
@@ -12,38 +12,82 @@
           actions, then generate this beauty.
         </p>
       </header>
-      <hr />
-      <h2 class="title">Mentors</h2>
+      <hr>
+
+      <!-- Mentors -->
+      <h2 class="title">
+        Mentors
+      </h2>
       <ul class="persons">
         <MentorCard
-          class="person"
-          v-for="mentor in mentors"
+          v-for="mentor in postList.mentor.items"
           :key="mentor.slug"
+          class="person"
           :mentor="mentor"
         />
       </ul>
-      <hr />
-      <h2 class="title">Mentees</h2>
+      <hr>
+
+      <!-- Mentees -->
+      <h2 class="title">
+        Mentees
+      </h2>
       <ul class="persons">
         <MenteeCard
-          class="person"
-          v-for="mentee in mentees"
+          v-for="mentee in postList.mentee.items"
           :key="mentee.slug"
+          class="person"
           :mentee="mentee"
         />
       </ul>
+      <client-only>
+        <infinite-loading @infinite="loadMoreMentees" />
+      </client-only>
     </div>
   </div>
 </template>
 
 <script>
 export default {
-  async asyncData({ $content, params }) {
```

Comment:
asyncData'yi ucurunca ssr destegini kaybediyor olabilir miyiz double check edebilir misin

Reply:
![ssrcheck](https://user-images.githubusercontent.com/22690563/101611800-035ebb00-3a1b-11eb-9bf4-999ff688c2ba.png)
İlgili bileşenler için SSR Özelliği bu değişimle birlikte ilk fetch aşamasındaki limit kadar hala sağlanıyor.

Fetch hook ile ilgili detaylı makale: https://nuxtjs.org/blog/understanding-how-fetch-works-in-nuxt-2-12/